.github/workflows/validate-pipelines.yml

```diff
@@ -0,0 +1,62 @@
+name: Validate Deployment Pipelines
+
+on:
+    pull_request:
+    push:
+        branches: [main, test-me-*]
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+    cancel-in-progress: true
+
+jobs:
+    files-changed:
+        name: files-changed
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+        # Map a step output to a job output
+        outputs:
+            gocd: ${{ steps.changes.outputs.gocd }}
+        steps:
+          - uses: actions/checkout@v3
+          - name: Check for relevant file changes
+            uses: getsentry/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
+            id: changes
+            with:
+              filters: |
+                gocd:
+                  - 'gocd/**'
+                  - '.github/workflows/validate-pipelines.yml'
+
+    validate:
+        if: needs.files-changed.outputs.gocd == 'true'
+        needs: files-changed
+        name: Validate GoCD Pipelines
+        runs-on: ubuntu-latest
+
+        # required for google auth
+        permissions:
+            contents: "read"
+            id-token: "write"
+
+        steps:
+            - uses: actions/checkout@v3
+            - id: 'auth'
+              uses: google-github-actions/auth@v1
+              with:
+                workload_identity_provider: 'projects/868781662168/locations/global/workloadIdentityPools/prod-github/providers/github-oidc-pool'
+                service_account: 'gha-gocd-api@sac-prod-sa.iam.gserviceaccount.com'
+                token_format: 'id_token'
+                id_token_audience: '610575311308-9bsjtgqg4jm01mt058rncpopujgk3627.apps.googleusercontent.com'
+                id_token_include_email: true
+            - uses: getsentry/action-gocd-jsonnet@v1
+              with:
+                jb-install: true
+                jsonnet-dir: gocd/templates
+                generated-dir: gocd/generated-pipelines
+            - uses: getsentry/action-validate-gocd-pipelines@v1
+              with:
+                configrepo: launchpad__main
```

Comment:
CI is failing because of this value, unclear if it's correct and isn't defined elsewhere or if here it should be a different value